src/planwise/client/dialog.cljs

```diff
@@ -1,11 +1,12 @@
 (ns planwise.client.dialog
   (:require [planwise.client.ui.rmwc :as m]
+            [re-frame.core :refer [dispatch]]
             [planwise.client.utils :as utils]))
 
 (defn new-dialog
```

Comment:
We might want to rename this component if it's used for other purposes other than creating a new thing. In particular if it's used for deleting such things ;)